tasks/task266_paper_reviews_classification.json

```diff
@@ -0,0 +1,2425 @@
+{
+    "Contributors": "Mihir Parmar",
+    "Source": "Paper Reviews Dataset (dataset link- https://archive.ics.uci.edu/ml/datasets/Paper+Reviews)",
+    "Categories": "Classification",
+    "Definition": "In this task, you are given a paper review, and we want you to evaluate the paper into one of the five evaluation metrics: (1) very negative, (2) negative, (3) neutral, (4) positive, and (5) very positive. This attribute (evaluation metric) represents the real evaluation given to the paper by the reviewers.",
```

Comment:
"evaluate the paper" or evaluate the sentiment of the review towards the paper?

Reply:
See this paper - Table 2 (https://sentic.net/wisdom2017fuentes.pdf)

Here, evaluate the paper means that according to the reviewer, how this paper content is on the scale of very negative - to - very positive. I think we can not say it sentiment.

What do you think @danyaljj ?